src/main/java/nl/beehivejava/scriptj/load/PluginLoader.java

```diff
@@ -0,0 +1,13 @@
+package nl.beehivejava.scriptj.load;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * @author Lesley
+ */
+public interface PluginLoader {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Annotate the "PluginLoader" interface with the @FunctionalInterface annotation (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1609)